node/src/service.rs

```diff
@@ -239,6 +240,7 @@ where
 	let prometheus_registry = parachain_config.prometheus_registry().cloned();
 	let transaction_pool = params.transaction_pool.clone();
 	let import_queue_service = params.import_queue.service();
+	// TODO there should be more here:
```

Comment:
Let's be more specific the extra configuration that you have spotted here, what are they related to.